### PR TITLE
feat: removing the bg color to the table of contents widget

### PR DESCRIPTION
**Pull request by TomDowling**
## Changes
Removed the bg color on the table of contents widget to bring it inline with the other widgets

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1476 #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
Looks great!

**Review by rebelchris — COMMENTED** [review]
Would you mind providing a screenshot, as for me locally it shows like this.
Not sure it's the same.

Screenshot 2023-06-26 at 11 57 42

**Comment by TomDowling** [comment]
> Would you mind providing a screenshot, as for me locally it shows like this. Not sure it's the same.
> 
> Screenshot 2023-06-26 at 11 57 42

Attached are updated screenshots for both use cases, collapsable and not, not the bg still shows on hover over the collapsable version, @rebelchris should i leave this as it is or remove that as well?
Screenshot 2023-06-27 at 7 40 08 am
Screenshot 2023-06-27 at 7 40 25 am
Screenshot 2023-06-27 at 7 40 32 am

**Comment by rebelchris** [comment]
@TomDowling Think the hover is fine as it's an action indicator.
Individual lines also have hover.

Weird I'm not seeing the border locally, but let's go with it :)

File: packages/shared/src/components/widgets/PostToc.tsx [hunk 1]
```diff
@@ -52,12 +53,12 @@ export default function PostToc({
   );
 
   const titleClass =
-    'flex items-center py-3 px-4 typo-body text-theme-label-tertiary';
-  if (collapsible) {
+    'flex items-center py-3 px-4 typo-body text-theme-label-tertiary border border-theme-divider-quaternary';
+  if (!collapsible) {
```
Comment (rebelchris) [inline packages/shared/src/components/widgets/PostToc.tsx:57]: Was this intended or for testing purpose?

**Review by rebelchris — APPROVED** [review]
Just one question, otherwise good to go.

File: packages/shared/src/components/widgets/PostToc.tsx (hunk 1 above)
Comment (TomDowling) [reply packages/shared/src/components/widgets/PostToc.tsx]: Good spot, was for the screenshot. Removed now

Comment (rebelchris) [reply packages/shared/src/components/widgets/PostToc.tsx]: Feel free to merge :)

**Pull request merged**